pages/1.13/security/ent/secure-compute-profiles/index.md

```diff
@@ -0,0 +1,77 @@
+
+For information about the configuration parameters used to enable secure computing mode on DC/OS clusters, see the [Configuration Reference](/1.13/installing/production/advanced-configuration/configuration-reference/).
+
+<!-- Leaving out next sections as too low-level per Andrei Budnik 
+
+# Secure computing isolator and filters
+DC/OS provides support secure computing mode on Linux through the `linux/seccomp` isolator and seccomp filters. The DC/OS seccomp isolator for Linux implements filters by grouping system calls into a set of commands that are specifically denied (blacklist) and a set of command that are explicitly allowed (whitelist).
+
+Seccomp filters reduce the attack surface of the Linux kernel by providing a mechanism for filtering of certain system calls. If you have agent nodes in the cluster with the Linux operating system, kernel version 3.5, or newer, you can use seccomp filters for containers launched using the DC/OS Universal Container Runtime (UCR) containers.
+
+Seccomp filters are defined in seccomp profiles using JSON formatting similar to this [sample seccomp profile](http://mesos.apache.org/documentation/latest/examples/seccomp_default.json).
+
+To support secure command execution for both UCR and Docker containers, the seccomp profile must include both the `pivot_root` system call and the `chroot` system call, in the `names` array. For example:
+
+```bash
+{
+  "names": [
+    "chroot",
+    "pivot_root"
+  ],
+  "action": "SCMP_ACT_ALLOW",
+  "args": [],
+  "comment": "",
+  "includes": {
+    "caps": [
+      "CAP_SYS_CHROOT"
+    ]
+  },
+  "excludes": {}
+},
+```
+
+# Configuring the agent for secure computing mode
+You can add the DC/OS seccomp isolator `linux/seccomp` to an agent by setting the `--isolation` or `--mesos-seccomp-enabled` agent configuration option. You must have `root` privileges to install the seccomp isolator and seccomp filters to avoid the issues involved when using a user account without root-level privileges.
+
+In addition to the `--isolation` or `--mesos-seccomp-enabled` option, you should configure the agent using the following settings:
+- Use `--seccomp_config_dir` to specify the path to the directory containing seccomp profiles.
+- Use `--mesos_seccomp_profile_name` to specify the default seccomp profile you want to apply by default for all DC/OS containers. The profile name must be relative to the `--seccomp_config_dir` path. If you don't specify this option, no default seccomp profile is defined and, therefore, no profile is applied by default.
+
+The following illustrates how to start a DC/OS agent with secure computing mode enabled:
+
+```bash
+sudo mesos-agent --master=<master ip> --ip=<agent ip> --work_dir=/var/lib/mesos --isolation=linux/seccomp --seccomp_config_dir=/etc/mesos/seccomp --seccomp_profile_name=default.json
```

Comment:
these instructions are for `mesos` NOT for DC/OS.   If you were going to start a mesos-agent this is what you would do.   THIS IS NOT the instructions on how to configure DC/OS which is what this document should be providing.